sql/pgwire_test.go

```diff
@@ -291,3 +294,58 @@ func TestPGPrepared(t *testing.T) {
 		}
 	}
 }
+
+func startInsecureServer(t *testing.T) (addr url.URL, done func()) {
+	ctx := server.NewTestContext()
+	ctx.Insecure = true
+	s := setupTestServerWithContext(t, ctx)
+
+	pgUrl := url.URL{
+		Scheme:   "postgres",
+		Host:     s.PGAddr(),
+		RawQuery: "sslmode=disable",
+	}
+
+	return pgUrl, func() {
+		cleanupTestServer(s)
+	}
+}
+
+func TestPGPython(t *testing.T) {
+	defer leaktest.AfterTest(t)
+
+	pgUrl, cleanupFn := startInsecureServer(t)
+	defer cleanupFn()
+
+	host, port, err := net.SplitHostPort(pgUrl.Host)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	c := exec.Command("docker",
```

Comment:
This test could go into `acceptance`, but I think we're going to want more tests than this for other languages (e.g. do we support the SQL generated by an out-of-the-box Django app?), so it may be better to figure out a good way to test things that live in other repos (and in the natural form for their language instead of string literals in a .go file). I started some tests in the [https://github.com/cockroachdb/cockroach-python](cockroach-python) repo using docker-compose. The current contents of that repo are obsolete but I had a start to some higher-level tests in bdarnell/cockroach-python@e1fad2f57eb25.